Pin electron version

akosyakov opened the pull request:
I've disabled auto format on save it breaks imports.

marcdumais-work commented:
@akosyakov about auto formatting breaking imports: can you give more details? 
I have recently seen something that might be related: upon saving a file, a bunch of my imports were removed at the top. By selectively disabling vs code extensions, I found that, in my case, removing "TypeScript Hero" fixed the issue.

akosyakov commented:
I've reverted it from the PR and added the following to my user settings by the advice of @jbicker : 
```json
    "typescriptHero.resolver.disableImportsSorting": true,
    "typescriptHero.resolver.organizeOnSave": false
```

marcdumais-work commented:
I'll try those settings, thanks
